lib/api/entities.rb

```diff
@@ -251,9 +251,18 @@ class ProjectService < Grape::Entity
       expose :push_events, :issues_events, :merge_requests_events, :tag_push_events, :note_events
       # Expose serialized properties
       expose :properties do |service, options|
-        field_names = service.fields.
-          select { |field| options[:include_passwords] || field[:type] != 'password' }.
-          map { |field| field[:name] }
+        field_names = []
+        service.fields.each do |field|
+          if field[:type] == 'fieldset'
+            field[:fields].each do |subfield|
+              if options[:include_passwords] || subfield[:type] != 'password'
```

Comment:
After going about it for half a day... 
The entities.rb contain GrapeEntity classes for which I can't find how to access the lib/api/helpers.rb defined helper method. If that is possible can you please let me know how to do so and I can implement it.